src/api/worker/WorkerImpl.ts

```diff
@@ -215,28 +215,39 @@ export class WorkerImpl implements NativeInterface {
 			get restInterface() {
 				return locator.cache
 			},
+
 			get serviceExecutor() {
 				return locator.serviceExecutor
 			},
+
 			get cryptoFacade() {
 				return locator.crypto
 			},
+
 			get cacheStorage() {
 				return locator.cacheStorage
 			},
+
 			get random() {
 				return {
 					async generateRandomNumber(nbrOfBytes: number) {
 						return random.generateRandomNumber(nbrOfBytes)
 					},
 				}
 			},
+
 			get eventBus() {
 				return locator.eventBusClient
 			},
+
 			get entropyFacade() {
 				return locator.entropyFacade
 			},
+
+			get workerFacade() {
+				// just creating it inline because no one on the worker side actually needs it so no reason to keep it around
+				return new WorkerFacade()
```

Comment:
may bite us when/if we ever add state, either rename to newWorkerFacade or store instance.